macros/contexts/contextUnits.pl

```diff
@@ -377,6 +382,43 @@ =head2 Working with numbers with units
 quantity is the absolute value of the original quantity.
 
 
+=head2 Differentiation of numbers with units
+
+In order to be able to differentiate a formula that returns a number
+with units, the MathObjects library needs to know the units of the
+variable you are differentiating by.  For example, if you have
+
+    $s = Compute("(3t^2 - 2x) m");
+
+as a function of time, C<t>, then you would like
+
+    $v = $s->D("t");
+
+to be equivalent to C<Compute("(6t - 2) m/s")>.
+
+To enable this, you must tell the C<Units> context that C<t> has units
+of seconds.  That is done using the C<asignUnits()> function of the
+context:
+
+    Context("Units")
+      ->withUnitsFor("length", "time")
+      ->assignUnits(t => "s");
+
+You can pass as many unit assignments to a single C<assignUnits()>
+call as you like.  E.g.
+
+    Context("Units")
+      ->withUnitsFor("length", "time")
+      ->assignUnits(
+        t => "s",
+        s => "m"
+      );
```

Comment:
Also, with this example, I think adding `->variables->are(t => Real, s=> Real);` will help.

Reply:
Never mind. I guess the `assignUnits` will defined the variables.

Reply:
I've added a sentence to the documentation indicating that that's what happens.